### ISSUE TITLE
Papers Citing Seed Papers not working in online versions

### ISSUE THREAD

**Issue by austindowney** [CitationGecko/gecko-react#104]
First, great project. I know (https://www.citationgecko.com/) is not being maintained anymore but wanted to ask the questions before I went down the path of trying to get this to work locally. 

On both sites 
- https://www.citationgecko.com/ 
- https://citationgecko.azurewebsites.net/ 

when I select a paper I can see what papers the seeded paper cites, but when I select _Papers Citing Seed Papers_ no papers showed up. This feature was working in July 2022 but stopped working around August 2022.

**Comment by ollings** [comment]
Yes, I would be great if there is an easy fix for this! Citation Gecko was my favorite literature search tool.

**Comment by brugarolaspablo** [comment]
Hi, this is such a great project. Could anyone provide an easier step-by-step tutorial on how to run Citation Gecko locally? 
? 

I have never built a React app... Maybe include screenshots illustrating the process. Thank you so much in advance! 

I hope papers citing seed papers is eventually fixed at the online version.
